src/screens/AccountList.js

```diff
@@ -58,37 +57,12 @@ export default class AccountList extends React.PureComponent {
 
 class AccountListView extends React.PureComponent {
   static propTypes = {
-    accounts: PropTypes.arrayOf(
-      PropTypes.shape({
-        address: PropTypes.string.isRequired,
-      })
-    ).isRequired,
+    accounts: PropTypes.object.isRequired,
     onAccountSelected: PropTypes.func.isRequired,
   };
 
   constructor(props) {
     super(props);
-    this.scrollToIndex = this.scrollToIndex.bind(this);
-  }
-
-  scrollToIndex() {
```

Comment:
This is useless IMO, and barely working. Removing it (it was playing with accountIds as well)